### PR TITLE
Feature/add ideas

### PR DESCRIPTION
**Pull request by darlaevans2000**
## Is this a fix or a feature?
fix
## What is the change?
css 
## What does it fix?
fixes the boxes, nav area, and text boxes
## Where should the reviewer start?
css file
## How should this be tested?
running index.html

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -157,8 +170,10 @@ bottom of card/show ideas button: #5356A4
   background-color: #5356A4;
   color: #E9E9F3;
   display: flex;
+  position: sticky;
```
Comment (BigBike96) [inline styles.css:173]: sticky! cool!

File: styles.css [hunk 2]
```diff
@@ -172,16 +187,19 @@ bottom of card/show ideas button: #5356A4
   color: #1F1F3D;
   padding-left: 20px;
   padding-right: 13px;
-  word-wrap: break-word;
+  overflow: scroll;
```
Comment (BigBike96) [inline styles.css:190]: nice scroll bar

File: index.html [hunk 3]
```diff
@@ -15,25 +15,25 @@ <h6 class="filter-label">Filter Starred Ideas</h6>
       <button class="show-button">Show Starred Ideas</button>
     </div>
   </nav>
-  <section>
+  <section class="main-wrap">
     <form class="top">
       <div class="title-group">
        <label class="input-label">Title</label>
-       <input class="title-input"></input>
+       <textarea rows="1" cols="78" id="titleInput" class="title-input"></textarea>
```
Comment (BigBike96) [inline index.html:22]: interesting way to deal with the input box size

**Review by BigBike96 — APPROVED** [review]
great styling with flexbox

**Review by rhen92 — APPROVED** [review]
Looks so good!! I now know to stick with percentages for future projects to make the website adjust based on the screen!

**Pull request merged**